doc/release-notes.md

```diff
@@ -35,6 +40,77 @@ If paired with a ziti controller/routers which support terminator strategies for
   * Handle listen to multiple edge routers.
   * Allow configuring max number of connections to edge routers
 
+# Release 0.13.6
+## Theme
+
+  * Fixes the `-n` flag being ignored for `ziti-enroll`
+
+# Release 0.13.5
+## Theme
+
+  * Adds ability to verify 3rd party CAs via the CLI in the Ziti Edge API
+
+## Ziti CLI Verify CA Support
+
+Previous to this version the CLI was only capable of creating, editing,
+and deleting CAs. Verification of a CA to make it useful for auto CA
```

Comment:
this is wordy imo:
"Verification of a CA to make it useful for auto CA
enrollment and authentication required manually created HTTP requests
and `openssl` signing commands to create a verification cert."

maybe:
The act of verifying newly created CAs was cumbersome and required manually created HTTP requests
and using `openssl` signing commands to create a verification cert

Reply:
It sure was. I was already fixing it.